### DISCUSSION TITLE
Custom Dataset model testing with mobilenet_v2 issue

### DISCUSSION THREAD

**Discussion by payal211** [open-mmlab/mmclassification#1544] [General] [answered]
Hi,

I have trained mobilenet_V2 on custom dataset. when I am testing the model result is as below:

Loads checkpoint by local backend from path: ./mobilenet_v2_custom_05_05_2023/epoch_190.pth
Inference ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
{
  "pred_label": 0,
  "pred_score": 0.9221256971359253,
  "pred_class": "tench, Tinca tinca"
}

I trained model on 6 classes. so pred_label and pred_score is good but the Pred_label is not taking the correct classes as per my classes. and it is showing the pre-defined ImageNet classes.

Can you please help me to load correct pred_classes as per actual classes.

Thanks

**Comment by Ezra-Yu** [comment, marked as answer]
you can try:
```
>>> from mmpretrain import ImageClassificationInferencer
>>> inferencer = ImageClassificationInferencer(
            model='configs/your_mobilenet_v2.py',
            weights='YOUR_CHECKPOINT_PATH.pth',
            device='cuda',
            classes=['A', 'B', C'.....])
>>> inferencer(['demo/dog.jpg', 'demo/bird.JPEG'], show_dir="./visualize/")
```

**Reply by payal211** [reply]
@Ezra-Yu Thanks for the quick response. 👍
